.github/workflows/cd.yml

```diff
@@ -1,32 +1,29 @@
-name: BUSINESS FILINGS UI CD
+name: Business Filings UI CD
 
 on:
   push:
     branches:
       - main
+      - feature*
+      - hotfix*
   workflow_dispatch:
     inputs:
-      environment:
-        description: "Environment"
+      target:
+        description: "Deploy To"
         required: true
         type: choice
         options:
         - dev
         - test
         - sandbox
         - prod
-      tagname:
-        description: "Specify a previous version (git tag) to deploy"
-        required: false
-        default: ""
 
 jobs:
   business-filings-cd:
-    uses: bcgov/bcregistry-sre/.github/workflows/ui-cd-node20.yaml@main
+    uses: bcgov/bcregistry-sre/.github/workflows/frontend-cd.yaml@main
     with:
-      environment: ${{ inputs.environment }}
-      tagname: ${{ inputs.tagname }}
+      target: ${{ inputs.target }}
+      app_name: "business-filings-ui"
```

Comment:
By the way, this app_name has to match a section in 1Password (CD vault), ie:

![image](https://github.com/user-attachments/assets/0679877c-b35e-4cfe-a925-9e61e70a3a38)